docs/basic-features/data-fetching.md

```diff
@@ -185,9 +185,9 @@ You should use `getStaticProps` if:
 For TypeScript, you can use the `GetStaticProps` type from `next`:
 
 ```ts
-import { GetStaticProps } from 'next'
+import { GetStaticProps, GetServerSidePropsContext } from 'next'
 
-export const getStaticProps: GetStaticProps = async (context) => {
+export const getStaticProps: GetStaticProps = async (context: GetServerSidePropsContext) => {
```

Comment:
You're using `GetServerSidePropsContext` on `getStaticProps` 🤔

Reply:
You are right, I've updated it 🤓